pkg/wwan/usr/bin/wwan-init.sh

```diff
@@ -707,6 +686,31 @@ event_stream | while read -r EVENT; do
   unset LOC_TRACKING_LL
   RADIO_SILENCE="$(parse_json_attr "$CONFIG" "\"radio-silence\"")"
   VERBOSE="$(parse_json_attr "$CONFIG" "\"verbose\"")"
+  QUERY_VISIBLE_PROVIDERS="$(parse_json_attr "$CONFIG" "\"query-visible-providers\"")"
+
+  if [ "$EVENT" = "PROBE" ]; then
+    PROBE_ITER="$((PROBE_ITER+1))"
+    # Every 20 seconds check the modem connectivity status.
+    # Quick probe only checks the status as reported by the modem,
+    # without generating any traffic.
+    EVENT="QUICK-PROBE"
+    if [ "$((PROBE_ITER % 15))" = "0" ] || [ "$STATUS_OUTDATED" = "y" ]; then
+      # Every 5 minutes update status.json.
+      # Also when QUICK-PROBE changes modem status (e.g. reconnects), next PROBE
+      # will be elevated to at least STANDARD-PROBE level.
+      # First update is not done immediately but after 5 minutes (PROBE_ITER starts with 1).
+      EVENT="STANDARD-PROBE"
+    fi
+    if [ "$QUERY_VISIBLE_PROVIDERS" = "true" ]; then
+      if [ "$((PROBE_ITER % 180))" = "31" ] || [ "$ENFORCE_LONG_PROBE" = "y" ]; then
```

Comment:
Am I right it makes to run the long probe every 1 hour but with a delay of 10 minutes from the beginning?
But why 31, not just 30?.. Tricky.

Reply:
Yes, correct.
For 31 vs. 30 see the comment in the code below this line, i.e:
```
# Every 1 hour additionally query the set of visible providers.
# Also after processing config change, next PROBE will be elevated to LONG-PROBE level.
# First LONG-PROBE is done after 10 minutes (modulo equals 31; PROBE_ITER starts with 1).
```
So PROBE_ITER is already at `1` with the first iteration. Then we add 30, because 30 * [20](https://github.com/milan-zededa/eve/blob/disable-visible-providers/pkg/wwan/usr/bin/wwan-init.sh#L32) = 600 seconds, i.e. 10 minutes.